### PR TITLE
update changelog for M60

### PR DESCRIPTION
**Pull request by chliangGoogle**

### CODE REVIEW

File: Firebase/Messaging/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,6 @@
+# 2019-11-19 -- v4.1.9
+- [changed] Move reliable message queue delete operation in a sync queue to avoid race conditions in unit tests. (#4236)
```
Comment (morganchen12) [inline Firebase/Messaging/CHANGELOG.md]: nit: replace `operation in a sync queue` -> `operations to a serial queue`

Comment (chliangGoogle) [reply Firebase/Messaging/CHANGELOG.md]: Done.

File: Firebase/Messaging/CHANGELOG.md [hunk 2]
```diff
@@ -1,3 +1,6 @@
+# 2019-11-19 -- v4.1.9
+- [changed] Moved reliable message queue delete operation to a serial queue to avoid race conditions in unit tests. (#4236)
```
Comment (paulb777) [inline Firebase/Messaging/CHANGELOG.md]: Should "reliable" be here?   It seems like it's mixing a feature description adjective with a description of a fix.

Comment (chliangGoogle) [reply Firebase/Messaging/CHANGELOG.md]: Done.

**Pull request merged**